src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -251,8 +251,11 @@ export class AKSClusterSettingsComponent
   }
 
   private _updateExternalClusterModel(): void {
-    const indexPosition = 2;
-    const version = this.controlValue(Controls.KubernetesVersion)?.main;
+    let version = this.controlValue(Controls.KubernetesVersion)?.main;
+    const versionSplitArr = version?.split('.');
+    if (versionSplitArr?.length && !(versionSplitArr[2] > 0)) {
```

Comment:
im checking if patch version > 0 then send all the input (e.g if it 1.22.1 -> it will show 1.22.1
and if it 1.22.0 -> it will show 1.22)
so if patch version > 0 then don't go inside the if statement and version will be the same as it is in the field

Reply:
BTW you can also use `patch` method of `semver` package that we have in our `package.json` instead of manual parsing. Something for the future perhaps.